src/types/complex-types/model.ts

```diff
@@ -56,7 +56,6 @@ import {
   assertArg,
   FunctionWithFlag
 } from "../../internal"
-import { ComputedValue } from "mobx/dist/internal"
```

Comment:
**praise:** @thegedge - it's very exciting to me that we can remove this import, which feels like a code smell, importing from MobX internals directly.